authors/apps/articles/serializers.py

```diff
@@ -10,6 +10,10 @@ class ArticleSerializer(serializers.ModelSerializer):
     author = serializers.SerializerMethodField()
     # Uploads an image to the Cloudinary servers
     images = serializers.ImageField(default=None)
+    likes = serializers.PrimaryKeyRelatedField(many=True, read_only=True)
```

Comment:
Consider renaming this to something related to what it does perhaps `reactors_id` or `user_id`

Reply:
okay noted